opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/VisitVaccineUtilTest.java

```diff
@@ -21,69 +20,98 @@
 import org.smartregister.immunization.util.VaccineCache;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
+import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyList;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.atLeastOnce;
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.verify;
 import static org.powermock.api.mockito.PowerMockito.mockStatic;
 import static org.powermock.api.mockito.PowerMockito.when;
+import static org.smartregister.domain.AlertStatus.normal;
 
 @RunWith(PowerMockRunner.class)
 @PrepareForTest(ImmunizationLibrary.class)
 public class VisitVaccineUtilTest {
     @Rule
     public MockitoRule rule = MockitoJUnit.rule();
 
-    @Mock
-    private Map<String, VaccineCache> vaccineMapMock;
-
     @Before
     public void setUp(){
         mockStatic(ImmunizationLibrary.class);
     }
 
     @Test
     public void testGetAllVaccinesGetsAllWomanAndChild() {
-        when(vaccineMapMock.get(anyString()))
-                .thenReturn(new VaccineCache());
+        Map<String, VaccineCache> vaccineMapMock = mock(HashMap.class);
+        VaccineRepo.Vaccine womanVaccine = VaccineRepo.Vaccine.rubella1;
+        VaccineCache womanVaccineCache = new VaccineCache();
+        womanVaccineCache.vaccineRepo = Collections.singletonList(womanVaccine);
+
+        VaccineRepo.Vaccine childVaccine = VaccineRepo.Vaccine.measles1;
+        VaccineCache childVaccineCache = new VaccineCache();
+        childVaccineCache.vaccineRepo = Collections.singletonList(childVaccine);
+
+        when(vaccineMapMock.get(eq("woman"))).thenReturn(womanVaccineCache);
+        when(vaccineMapMock.get(eq("child"))).thenReturn(childVaccineCache);
+
         when(ImmunizationLibrary.getVaccineCacheMap())
                 .thenReturn(vaccineMapMock);
 
-        VisitVaccineUtil.getAllVaccines();
-        verify(vaccineMapMock).get(eq("woman"));
-        verify(vaccineMapMock).get(eq("child"));
+        Map<String, VaccineRepo.Vaccine> vaccinesMap = VisitVaccineUtil.getAllVaccines();
+        Assert.assertTrue(vaccinesMap.containsValue(womanVaccine));
+        Assert.assertTrue(vaccinesMap.containsValue(childVaccine));
```

Comment:
1. Assert the expected sizes of each
2. Assert that spaces and underscores are stripped